src/game/common/ini/ini.h

```diff
@@ -170,20 +170,22 @@ class INI
     void Unprep_File();
 
     File *m_backingFile;
-    char m_buffer[MAX_BUFFER_SIZE];
+    char m_buffer[INI_BUFFER_SIZE];
     int m_bufferReadPos;
     int m_bufferData;
     Utf8String m_fileName;
     INILoadType m_loadType;
     int m_lineNumber;
-    char m_currentBlock[MAX_LINE_LENGTH];
-    char m_blockEnd;
```

Comment:
I removed `m_blockEnd` and added one to `m_currentBlock`. This is much better logically.